Cargo.toml

```diff
@@ -6,23 +6,44 @@ members = [
 
 # The release profile, used for `cargo build`.
 [profile.dev]
-incremental = true
-opt-level = 0
+opt-level = 1
 debug = true
-rpath = false
-lto = false
 debug-assertions = true
 overflow-checks = true
+lto = false
 panic = 'unwind'
+incremental = true
+rpath = false
 
 # The release profile, used for `cargo build --release`.
 [profile.release]
-incremental = false
 opt-level = 3
 debug = false
-rpath = false
-codegen-units = 1
+debug-assertions = false
+overflow-checks = false
 lto = true
+panic = 'unwind'
+incremental = false
+codegen-units = 1
+rpath = false
```

Comment:
I have a feeling these can go as they're most likely the default anyway, same with opt-level, in fact which ones here are different from the default?

Reply:
Here we have the default profiles: https://doc.rust-lang.org/cargo/reference/profiles.html#default-profiles

I did maintain some of the defaults, just to make things more clear. I can remove redundant options if you prefer it :)

Reply:
Yeah I think it would be good to remove redundant options, when I see options here I assume we’ve changed them for a reason, so default ones seem odd.